source/util.cpp

```diff
@@ -73,6 +73,23 @@ string replace_(string const &s, string const & from, string const &to)
 	return r;
 }
 
+/// Move all occurances of non-alphanumerical characters to the end of the string and replace them with their ASCII codes.
+std::string to_alphanumerical(const std::string& input)
```

Comment:
please use the same style of `const` placement as rest of the code

Reply:
Oh, yes.